source/includes/_authenticated_api_members.md

```diff
@@ -27,6 +27,35 @@ Find a member by id. Member ids can be discovered via a call to the lookup API.
 `GET /api/v1/members/123`
 
 
+
+### Activity
+```json
+{
+  "first_name": "Jane",
+  "last_name": "Doe",
+  "phone_number: "555-555-5555",
+  "created_at": "2015-06-01T15:37:47Z",
+
+  "signatures": [
+    {
+      "email": "jane.doe@example.com",
+      "petition": {
+        "slug": "save-the-whales"
+      }
+      ...
+    }
+    ...
+  ]
+  ...
+}
+```
+
+Get information about a member's activity on the platform.
```

Comment:
Could you add a sentence or two explaining why this might be useful and what it includes?